spec/unit/blockscore/errors/not_found_error_spec.rb

```diff
@@ -1,6 +1,9 @@
 module BlockScore
   RSpec.describe NotFoundError do
     subject { -> { BlockScore::Person.retrieve('invalid_id') } }
+    let(:message) do
+      '(Type: invalid_request_error) Person with id invalid_id could not be found (Status: 404)'
```

Comment:
@backus one thing I noted here is that Not found errors still return a type of invalid_request_error. I was confused by that at first, but noted that it is aligned with the API docs, AND I confirmed it interactively against the live server. SO while the message says not found and status = 404 the type doesn't feel like it coincides. THis IS how it is presently implemented and that response IS from the backend.